generator/pubspec.yaml

```diff
@@ -10,14 +10,19 @@ dev_dependencies:
   source_gen_test: ^1.0.2
   test: ^1.20.1
 dependencies:
-  analyzer: ^3.2.0
+  analyzer: ^4.0.0
   build: ^2.2.1
   code_builder: ^4.1.0
   path: ^1.8.1
-  # smartstruct:
-  #   path: ../smartstruct
-  smartstruct: ^1.2.7
-  source_gen: ^1.2.1
-dependency_overrides:
+  #smartstruct:
+  #  path: ../smartstruct
+  #smartstruct: ^1.2.7
   smartstruct:
-    path: ../smartstruct
+    git:
```

Comment:
I think this is causing a problem with the GithubAction as the Action does not have access to clone your repository.
This should be reverted to the original smartstruct dependency I think

Reply:
Just committed the reversal of the `pubspec.yml`.